src/main/java/nextstep/qna/domain/Answer.java

```diff
@@ -72,6 +72,11 @@ public void toQuestion(Question question) {
         this.question = question;
     }
 
+    public DeleteHistory delete() {
+        setDeleted(true);
```

Comment:
```suggestion
        this.deleted = true;
```

위와 같이 구현하고 setter 메서드를 제거하는 것은 어떨까?